rpc_jobs/standard_job_postmerge.yml

```diff
@@ -2,14 +2,36 @@
     name:       "gating-post-merge"
     repo_name:  "rpc-gating"
     repo_url:   "https://github.com/rcbops/rpc-gating"
-    # `branch` defaults to `master` and can be omitted.
-    #branch:     "master"
+    branch:     "master"
     image:      "xenial"
     scenario:   "functional"
     action:     "test"
     jira_project_key: "RE"
+    trigger_build: "PM_{repo_name}-{branch}-{image}-{scenario}-{action}"
     jobs:
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+      - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+# This job is specified as `freestyle` as using a pipeline job with github
+# trigger causes the job to be triggered irrespective of which branch was
+# pushed to.
+- job-template:
+    name: "PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}"
+    project-type: freestyle
+    scm:
+      - git:
+          url: "{repo_url}"
+          branches:
+           - "{branch}"
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    triggers:
+        - github
+    builders:
+      - trigger-builds:
+        - project:
+            - "{trigger_build}"
 
 - job-template:
     name: 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
```

Comment:
I think this name should be changed as well, e.g. `PM-periodic-trigger_`. Having one called `PM_` and the other `PM-push-trigger_` makes for an awkward naming scheme that lacks clarity when reading the list of jobs.

Reply:
I think the difference is justified as the push trigger requires a separate job, but the timer/periodic trigger is part of the main job. 

Otherwise the push trigger job will trigger the periodic trigger job, which doesn't make so much sense to me.

Reply:
That's a fair point. I'm not familiar enough with this to suggest an alternative but these types of naming inconsistency tend to build over time and create a whole load of unnecessary knowledge that must be learnt. If it can be removed I think that would be preferable.

Reply:
I'm not swayed enough one way or another to suggest further edits -- but I am now mindful of increasingly long job names and for that am happy to keep the job name as it stands.